### PR TITLE
load 10 per page and load more as user scrolls

### PR DESCRIPTION
**Pull request by saito-sv**

### CODE REVIEW

File: components/Skeletons/pool/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { memo } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+import Skeleton from 'components/Skeletons';
+import Shimmer from 'components/Skeletons/Shimmer';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(3),
+    padding: '10px 15px',
```
Comment (prodvdc) [inline components/Skeletons/pool/index.js:10]: Can we use theme.spacing for padding?

**Review by prodvdc — APPROVED** [review]
LGTM! It works nice.

File: components/Skeletons/pool/index.js (hunk 1 above)
Comment (saito-sv) [reply components/Skeletons/pool/index.js:10]: how could that work?

Comment (saito-sv) [reply components/Skeletons/pool/index.js:10]: I mean, how does that looks like when theres 2 values to apply spacing to?

Comment (prodvdc) [reply components/Skeletons/pool/index.js:10]: e.g. theme.spacing(10 / 8, 15 / 8) or theme.spacing(1, 2)

**Comment by ghost** [comment]
Filters seems to be working fine but my pools don't show at top once I log into my account.

**Comment by ghost** [comment]
Will rebase this PR to develop and see how it's working

**Comment by ghost** [comment]
rebased & opened for review

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**